release.nix

```diff
@@ -125,21 +134,26 @@ let
 
   jobs = {
     inherit dockerImageArtifact;
-    native =
+    linux =
       let filteredBuilds = mapAttrsRecursiveCond (a: !(isList a)) (path: value:
         if (any (p: take (length p) path == p) onlyBuildOnDefaultSystem) then filter (s: !(elem s nonDefaultBuildSystems)) value else value)
         (packagePlatforms project);
       in (mapTestOn (__trace (__toJSON filteredBuilds) filteredBuilds));
+    darwin =
+      let filteredBuilds = mapAttrsRecursiveCond (a: !(isList a)) (path: value:
+        if (any (p: take (length p) path == p) onlyBuildOnDefaultSystem) then filter (s: !(elem s nonDefaultBuildSystems)) value else value)
+        (darwinRelease.packagePlatforms darwinRelease.project);
+      in (darwinRelease.mapTestOn (__trace (__toJSON filteredBuilds) filteredBuilds));
```

Comment:
Now that there are top-level attrsets per platform, the `.x86_64-darwin` suffix is redundant. How difficult would it be to remove that?

Reply:
This combined with the `linuxRelease` suggestion, makes me wonder if we should bite the bullet and remove use of `release-lib`.  I think it mostly just adds the suffix?

Reply:
Yes, I was thinking the same. The other thing that release-lib adds is difficulty.
The package platforms filtering should be factored out into a mini-release-lib however. We don't want to have the same jobset flitering code sprayed everywhere.